data_broker.js

```diff
@@ -302,7 +322,16 @@ class DataBroker extends EventTarget {
       }
 
       this.dispatchEvent(new CustomEvent('stats-stream-data', {
-        detail: this._update,
+        detail: {
+          stats: this._update,
```

Comment:
Can you revert this back to the way it was now that we're emitting the mapped read data in a separate event?

Reply:
Yes, I made a separate event to emit the mapped reads from the index file. In my last solution, I put them in the `stats-stream-data` together. Do you mean that I revert them back to:
`  this.dispatchEvent(new CustomEvent('stats-stream-data', {
        detail: {
          stats: this._update,
          mappedReads: mappedReads,
          unmappedReads: unmappedReads,
        }
      }));`

Or `  this.dispatchEvent(new CustomEvent('stats-stream-data', {
        detail: this._update 
}`

Reply:
I think we want this one: `this.dispatchEvent(new CustomEvent('stats-stream-data', { detail: this._update  }`